Resources/config/services.yml

```diff
@@ -31,3 +32,16 @@ services:
         class: %support_tools.info_collectors.php.ezc.class%
         arguments:
             - @support_tools.info_collector.system_info.ezc
+
+    support_tools.info_collectors.symfony.kernel.config:
+        class: %support_tools.info_collectors.symfony.kernel.config.class%
+        arguments:
+            - %kernel.environment%
```

Comment:
That's a lot of arguments. What about injecting `%kernel%` instead ? It would also simplify the collector.